scripts/Muon/fft_model.py

```diff
@@ -42,19 +41,22 @@ def execute(self):
         """
         runs the relevant parts of the FFT and the preprocessing
         """
-        if self.phaseTable is not None:
-            if self.phaseTable["newTable"]:
-                self.model.makePhaseQuadTable(self.phaseTable["axis"],self.phaseTable["Instrument"])
-            self.model.PhaseQuad()
+        try:
+            if self.phaseTable is not None:
+                if self.phaseTable["newTable"]:
+                    self.model.makePhaseQuadTable(self.phaseTable)
+                self.model.PhaseQuad()
 
-        if self.preRe is not None:
-            self.model.preAlg(self.preRe)
+            if self.preRe is not None:
+                self.model.preAlg(self.preRe)
 
-        if self.preIm is not None:
-            self.model.preAlg(self.preIm)
+            if self.preIm is not None:
+                self.model.preAlg(self.preIm)
 
-        if self.FFT is not None:
-            self.model.FFTAlg(self.FFT)
+            if self.FFT is not None:
+                self.model.FFTAlg(self.FFT)
+        except:
+            pass
```

Comment:
Would it not be better to show some kind of warning that an error occurred, as opposed to just failing silently? Or does this happen elsewhere?

Reply:
this is for when a user cancels the algorithm. It still says that a user cancelled it but removes the stack trace